Yoast/YoastSEO.js
# Place the measurement elements holder off screen.

## Summary

Move the measurement elements holder `<div>` off-screen so it doesn't break the responsive view when opening the media modal.

## Relevant technical choices:

I've initially considered to put the required CSS inline in the JS template but this would not allow to swap the `left` property for RTL languages. The CSS needs to be in a file that goes through the RTL files generation.
Also, added an `aria-hidden` attribute because we don't want this element content to be available for assistive technologies.

## Test instructions

- be sure the `yoastseo` module is "yarn linked"
- on wordpress-seo, build the CSS and JS
- emulate a responsive view (e.g. iPhone) using your browser's dev tools
- edit a post and click on "Add Media" to open the WordPress media modal
- ensure the media modal has a correct size and doesn't extend on the right; for reference see the screenshot below

On the left: good. On the right, bad:

![screen shot 2017-01-03 at 09 33 44](https://cloud.githubusercontent.com/assets/1682452/21603079/ccde8cbe-d19a-11e6-9438-5013e0233e3a.jpg)

Fixes #1006
